recipes-core/images/pantavisor-bsp.bb

```diff
@@ -57,6 +57,7 @@ fakeroot do_compile(){
     export PVR_DISABLE_SELF_UPGRADE=true
     if [ -d ${WORKDIR}/pv-developer-ca_${PVS_VENDOR_NAME} ]; then
         tar -C ${PVR_PVBSPIT_CONFIG_DIR}/ -xf ${WORKDIR}/pv-developer-ca_${PVS_VENDOR_NAME}/pvs/pvs.defaultkeys.tar.gz --no-same-owner
+        tar -C ${PVR_PVBSPIT_CONFIG_DIR}/ -xf ${WORKDIR}/pv-developer-ca_${PVS_VENDOR_NAME}/pvs/pvs.oemkeys.tar.gz --no-same-owner
```

Comment:
is oem truststore then mandatory? maybe this should be optional?

Reply:
maybe you are right but, aren't we going to always work with pv-developer-ca commit 4d0de8e53876f52b8b6a6dd8cab669a78e8dc896, which from now on will always have that tarball? (Maybe I'm missing some yocto thing)

Reply:
@anibalportero i think most products will have OEM, but i would not make it mandatory for now.

Reply:
this should be solved now with the current changes in classes/pvroot-image.bbclass